examples/dielectric.jl

```diff
@@ -0,0 +1,54 @@
+# Compute a few eigenvalues of the dielectric matrix (q=0,ω=0) iteratively
```

Comment:
Do you want to convert this into a "proper" example for the website?

Reply:
I don't think so; not unless we gain a bit more experience with it at least, and maybe generalize it to q, omega